src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
+import React from 'react';
+import HomePage from '../src/components/pages/HomePage/HomePage';
```

Comment:
Add entry point file js index.js to pages
```suggestion
import HomePage from '../src/components/pages/';
```